README.md

```diff
@@ -1 +1,104 @@
-# Metrics-WebApp
\ No newline at end of file
+<div align="center">
+
+  <h1><b>Metrics webapp</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## About the Project
+
+> This project is a React capstone project for Microverse. It is a mobile web application to check a list of metrics (numeric values) retrieved from a selected API.
+
+## Built With
+
+- HTML
+- Bootstrap 5
+- JavaScript
+- React
+- Redux
+- Jest
+- API
+
+### Key Features
+
+- Fetching and displaying country data from API
+- Filter out country category based on desired parameters.
+
+## Live Demo
+
+[Live Demo](https://metrics-web-app.onrender.com)
+
+## Getting Started
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-prefer-folder
+
+  git clone https://github.com/c00p75/Metrics-WebApp.git
+```
+
+### Run this command to install dependencies
+
+> npm install
+
+### Run this command to run server
+
+> npm start
+
+### Run this command to run test
+
+> npm run test
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **George M'sapenda**
+
+- GitHub: [c00p75](https://github.com/c00p75)
+- LinkedIn: [George M'sapenda](ttps://www.linkedin.com/in/georgemsapenda/?originalSubdomain=zm)
```

Comment:
- [ ] Kindly make your LinkedIn handle a link so one will click on it and is taken to your LinkedIn profile. 👍🏽